Lib/multiprocessing/context.py

```diff
@@ -223,6 +223,10 @@ class Process(process.BaseProcess):
     def _Popen(process_obj):
         return _default_context.get_context().Process._Popen(process_obj)
 
+    @staticmethod
+    def _after_fork():
```

Comment:
Yes, this method is necessary when the default context is `spawn` (which is the case on MacOS) and the calling code use the default context without explicitly specify a context. eg.,

```
import multiprocessing as mp

global_resource = mp.Semaphore()

def submain(): pass

if __name__ == '__main__':
    p = mp.Process(target=submain)
    p.start()
    p.join()
```

Without the above code, named resources still leak. Because the subprocess will use `_after_fork` from `BaseProcess`